launch:new-altcha-guide (#1810)
aziontech/docs: src/content/docs/en/pages/guides/edge-functions/altcha-challenge.mdx

- **`/az-request-verify`:** The endpoint that displays the CAPTCHA challenge. Users should be redirected here whenever a challenge is required. Accessing it returns a new CAPTCHA challenge for verification.
- **`/az-request-captcha`:** After a user successfully solves the challenge, the ALTCHA script automatically calls this endpoint to validate the solution. If validation succeeds, the script sets a session cookie in the user’s browser, allowing future requests to be verified automatically without new challenges.

### Workflow

1. The user accesses a protected endpoint in your application.
2. The security layer redirects the user to the `/az-request-verify` endpoint.
3. The CAPTCHA challenge is displayed to the user.
4. After solving the challenge, ALTCHA sets session cookies in the user’s browser.
5. The same user’s future requests are validated automatically via session cookies, without requiring new challenges.

---

## Configuring the integration

### Setting up an edge firewall

ALTCHA runs on the **Edge Firewall**. To enable it, first create a rule in Edge Firewall, following these steps:

1. Open Azion Console > **Edge Firewall**.
2. Click **+ Edge Firewall**.

> 3. Enter a memorable name for your edge firewall.

Author: ```suggestion
3. Give a name to your edge firewall.

```
